my_style.css

```diff
@@ -220,3 +220,88 @@ form p {
   text-decoration: underline;
   font-size: 20px;
 }
+
+.slideshow {
+  max-width: 600px;
+  position: relative;
+  margin: 30px auto;
+}
+
+.slideshow_img img {
+  width: 100%;
+  height: 400px;
+  object-fit: cover;
+  border-radius: 10px;
+  display: block;
+}
+
+#prev, #next {
+  cursor: pointer;
+  position: absolute;
+  top: 45%;
+  padding: 12px;
+  font-weight: bold;
+  font-size: 60px;
+  color: palevioletred;
+  text-shadow: 0px 0px 5px rgba(0, 0, 0, 0.7);
+  user-select: none;
+}
+
+#prev { left: -10%; }
+#next { right: -10%; }
+
+#prev:hover, #next:hover {
+  color: purple;
+}
+
+@media (max-width: 600px) {
+  .slideshow {
+    max-width: 90%;
```

Comment:
On a big screen this works:
<img width="901" height="542" alt="image" src="https://github.com/user-attachments/assets/851754d7-6249-4d4f-8c20-4d95ac24f908" />

On a small screen too:
<img width="291" height="463" alt="image" src="https://github.com/user-attachments/assets/0581ffc4-23f5-46ac-90ad-4a559b01dacb" />

But in-between, we loose the arrows:
<img width="722" height="547" alt="image" src="https://github.com/user-attachments/assets/6f971593-94ea-4e28-bb67-f85a99ed30c8" />